VimeoUpload/Upload/Extensions/AFURLSessionManager+Upload.swift

```diff
@@ -30,6 +30,6 @@ extension AFURLSessionManager
 {
     func uploadTaskForIdentifier(identifier: Int) -> NSURLSessionUploadTask?
     {
-        return self.uploadTasks.filter{ $0.taskIdentifier == identifier }.first as? NSURLSessionUploadTask
+        return self.uploadTasks.filter{ $0.taskIdentifier == identifier }.first as NSURLSessionUploadTask!
```

Comment:
Same here, we should revert this change so we simply return nil in the event of a cast failure